bee-common/bee-packable/src/error.rs

```diff
@@ -15,20 +15,9 @@ pub enum UnpackError<T, U> {
 }
 
 impl<T, U> UnpackError<T, U> {
-    /// Maps the [`Packable`](crate::Packable) variant of this enum.
-    pub fn map<V, F: Fn(T) -> V>(self, f: F) -> UnpackError<V, U> {
-        match self {
-            Self::Packable(err) => UnpackError::Packable(f(err)),
-            Self::Unpacker(err) => UnpackError::Unpacker(err),
-        }
-    }
-
-    /// Coerces the value by calling `.into()` for the [`Packable`](UnpackError::Packable) variant.
-    pub(crate) fn coerce<V>(self) -> UnpackError<V, U>
-    where
-        T: Into<V>,
-    {
-        self.map(|x| x.into())
+    /// Wraps an error in the [`Packable`](UnpackError::Packable) variant.
+    pub fn from_packable(err: impl Into<T>) -> Self {
```

Comment:
Doesn't this make more sense being called `into_packable`? In most of our uses, it's taking a `ValidationError` and converting to `UnpackError::Packable(MessageUnpackError)`

Reply:
my motivation to call this "from" instead of "into" is that you call it as `Unpack_error::from_packable` but I'm not really happy with the name either.